apps/01PigLatin.js

```diff
@@ -4,13 +4,53 @@ var assert = require('assert');
 var prompt = require('prompt');
 prompt.start();
 
-
 function pigLatin(word) {
 
-    // Your code here
+  //Attaches "yay" if word begins with vowel:
+  if(word[0] === 'a' || word[0] === 'e' || word[0] === 'i' || word[0] === 'o' || word[0] === 'u' ) {
+      return word + "yay";
+  }
```

Comment:
@dorianpenaloza You don't account for `y` here. This isn't consistent with your if statements below, where you do check for the vowel `y`.